roles/keystone-setup/tasks/main.yml

```diff
@@ -34,6 +34,7 @@
   keystone_user: tenant={{ item }}
                  tenant_description="{{ item }} tenant"
                  token={{ admin_token }}
+                 endpoint="{{ endpoints.keystone.url.port.admin }}/{{ endpoints.keystone.version }}/" 
```

Comment:
This variable endpoint is added since the we aren't using default ports of openstack